### PR TITLE
Fix(typography): overflow `code` tag style

### LINKED ISSUE #40261: Typography.Text code ellipsis 丢失右边框
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/s/sheng-lue-hao-antd-5-1-5-forked-1gog5y?file=/demo.tsx)

### Steps to reproduce

开启 ellipsis Switch

### What is expected?

Typography.Text code 有完整的边框
image

### What is actually happening?

Typography.Text code 丢失右边框
image

| Environment | Info |
| --- | --- |
| antd | 5.1.5 |
| React | 18.2.0 |
| System | macOS 12.4 |
| Browser | 108.0.5359.98 |

### PR DESCRIPTION
**Pull request by wangcch**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/40261

I think it is normal for the `code` tag to show the original size. We may just need to modify the style.🤔

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Fix overflow `<code />` style (Typography)  |
| 🇨🇳 Chinese |  修复 `<code />` 溢出样式 (Typography)  |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/typography/style/index.ts [hunk 1]
```diff
@@ -54,6 +54,17 @@ const genTypographyStyle: GenerateStyle<TypographyToken> = (token) => {
         userSelect: 'none',
       },
 
+      [`&${componentCls}-style-code`]: {
```
Comment (afc163) [inline components/typography/style/index.ts:57]:
```suggestion
      [`&${componentCls} > code`]: {
```

What about that?

Comment (wangcch) [reply components/typography/style/index.ts:57]: The class (`ant-typography-style-code`) is for `<span />`. not `<code />` 👀

image

File: components/typography/Base/index.tsx [hunk 2]
```diff
@@ -509,6 +509,7 @@ const Base = React.forwardRef<HTMLElement, BlockProps>((props, ref) => {
                 [`${prefixCls}-single-line`]: mergedEnableEllipsis && rows === 1,
                 [`${prefixCls}-ellipsis-single-line`]: cssTextOverflow,
                 [`${prefixCls}-ellipsis-multiple-line`]: cssLineClamp,
+                [`${prefixCls}-style-code`]: restProps.code,
```
Comment (afc163) [inline components/typography/Base/index.tsx:512]:
```suggestion
                [`${prefixCls}-code`]: restProps.code,
```

**Comment by wangcch** [comment]
Now there's a problem. The style may not be as expected. 😅


image

**Comment by yoyo837** [comment]
rebase please

**Comment by wangcch** [comment]
> rebase please

Done👌

**Pull request closed without merging**